src/sentry/static/sentry/app/actionCreators/globalSelection.jsx

```diff
@@ -0,0 +1,21 @@
+/*eslint no-use-before-define: ["error", { "functions": false }]*/
+
+import {isInteger} from 'lodash';
+import GlobalSelectionActions from 'app/actions/globalSelectionActions';
+
+/**
+ * Updates global project selection
+ *
+ * @param {Number[]} projects List of project ids
+ */
+export function updateProjects(projects) {
+  if (!isProjectsValid(projects)) {
+    return;
```

Comment:
We should capture this to sentry, and possibly show an error to the user if this happens, otherwise it will be an odd interaction if nothing happens after they select a project.

Reply:
Added logging to sentry. I don't think this can happen if a user is selecting a project via the dropdown since those are already validated.